src/components/Homepage/HeroSection.tsx

```diff
@@ -18,7 +18,7 @@ const HeroSection: React.FC = () => {
         {/* Call to Action Buttons */}
         <div className="flex flex-col md:flex-row items-center justify-center space-y-4 md:space-y-0 md:space-x-6">
           <a
-            href="#explore"
+            href="https://ajay-dhangar.github.io/algo/docs/"
```

Comment:
Change the link in the "Explore Algorithms" button from "https://ajay-dhangar.github.io/algo/docs/" to "/docs/".

Reply:
I think that should be /algo/docs as /algo/ is root